DataServer/email_sender.py

```diff
@@ -0,0 +1,59 @@
+import smtplib
+from email.message import EmailMessage
+class email_sender:
+    
+    def __init__(self, logger, subject=None, body=None, configDict=None) -> None:
```

Comment:
Should an email sender be created every single time you want to send an email?  Or should it exist and be able to send multiple emails?